src/telegram_bot.py

```diff
@@ -55,27 +61,33 @@ async def get_f1_fantasy_standings(
                 url=league_url,
                 headers={"Cookie": cookies},
             )
-            standing_message = get_standings_as_string(req)
-            # FIXME: At the moment the parse_mode has been set to MARKDOWN.
-            # We have to decide how to show the standings
-            await context.bot.send_message(
-                chat_id=update.effective_chat.id,
-                text=escape_markdown(f"```{standing_message}```", version=2),
-                parse_mode=ParseMode.MARKDOWN_V2,
-            )
+            standings = decode_http_response(req, to_league_standings)
+
+            if isinstance(standings, LeagueStanding):
+                # TODO: We can improve this part as well
+                table = pt.PrettyTable(["Username", "Points"])
+                for entrant in standings.entrants:
+                    table.add_row([entrant.user.username, entrant.score])
+                await context.bot.send_message(
+                    chat_id=update.effective_chat.id,
+                    text=escape_markdown(f"{table}", version=2),
+                    parse_mode=ParseMode.MARKDOWN_V2,
+                )
+            else:
+                await context.bot.send_message(
+                    chat_id=update.effective_chat.id,
+                    text="Non è stato possibile recuperare la classifica",
+                )
 
         return get_f1_fantasy_standings
 
 
-# TODO Define the format of the output message
-def get_standings_as_string(req: Union[Response, Response]) -> str:
+# TODO Move to utility package/file
+def decode_http_response(
```

Comment:
This method could be used to parse every HTTP requests for this project (of course we can improve how it works).

The idea is that every time we have to parse a JSON coming from an HTTP request, we are going to check the status code, and then, convert the JSON into an object. If we have multiple handlers, we are going to duplicate every time this logic.

So, this method can help us to deal with that: it accept an HTTP request and a parameter called `decode_fn` which is a function that converts a `dict` to a parameter. 
Looking at the implementation, the `decode_http_response` applies the `decode_fn` function in case of 200 status code.

Examples:

```
def convert_to_string(input: dict) -> string:
    return str(input["key"])

# Assume the Animal class exists
def convert_to_animal(input: dict) -> Animal:
    return Animal(key=input["key"])

req = requests.... # HTTP request here
my_string: str = decode_http_response(req, convert_to_string)

my_animal: Animal = decode_http_response(req, convert_to_animal)
```